client/plots/sampleScatter.js

```diff
@@ -655,31 +639,27 @@ class Scatter {
 			const origin = morigin[mutation.origin]?.label
 			const dtlabel = origin ? `${origin[0]} ${dt2label[dt]}` : dt2label[dt]
 
-			G
-			.append('text')
-			.attr('x', offsetX)
-			.attr('y', 25)
-			.text(dtlabel)
-			.style('font-weight', 'bold')
-			.style('font-size', '0.8em')
+			G.append('text')
+				.attr('x', offsetX)
+				.attr('y', 25)
+				.text(dtlabel)
+				.style('font-weight', 'bold')
+				.style('font-size', '0.8em')
 
 			offsetY = 50
 			for (const [key, category] of map) {
 				const assay = key.split(',')[1]
-				if(key.includes(dtlabel))
-				G
-				.append('text')
-				.attr('x', offsetX)
-				.attr('y', offsetY)
-				.text(`${category.sampleCount}${category.hasOrigin?assay[0]: ''}`)
-				.style('font-size', '0.8em')
+				if (key.includes(dtlabel))
+					G.append('text')
+						.attr('x', offsetX)
+						.attr('y', offsetY)
+						.text(`${category.sampleCount}${category.hasOrigin ? assay[0] : ''}`)
+						.style('font-size', '0.8em')
 				offsetY += 25
 			}
 			offsetX += step
-
 		}
 		return offsetY
-		
 	}
 
 	hideCategory(G, tw, key, hide) {
```

Comment:
this one too. is G a svg element?

Reply:
this line was added because of reformatting, so not related to this PR. It is an svg group, that is called g in svg.